python/kserve/kserve/model_server.py

```diff
@@ -181,9 +185,20 @@ def __init__(
             enable_grpc: Whether to turn on grpc server. Default: ``True``
             enable_docs_url: Whether to turn on ``/docs`` Swagger UI. Default: ``False``.
             enable_latency_logging: Whether to log latency metric. Default: ``True``.
-            configure_logging: Whether to configure KServe and Uvicorn logging. Default: ``True``.
+            configure_logging: Whether to configure KServe and Uvicorn logging. Default: ``None``.
+                               ``Deprecated``: This argument is deprecated since, 0.13.0. Use standard
+                               function 'configure_logging' from 'kserve.logging' module and explicitly set
+                               'log_config' arg to 'None' or use --configure_logging cmd arg to disable logger
+                               configuration.
```

Comment:
This doc should be moved to the command line arg parser so it is exposed to user when running the model server

Reply:
After discussing with @yuzisun, we decided to remove these parameters from model_server